catalog/ui/catalog-ui-search/src/main/webapp/component/filter/filter.view.js

```diff
@@ -67,10 +67,12 @@ define([
             case 'DWITHIN':
                 break;
             default:
-                if (value[0].constructor === Object) {
+               if (value[0] == null) {
```

Comment:
Why do we need a null check here?

Reply:
The response coming back from the `Search Form` backend won't populate values in some cases, since it is a template.  This is currently being reworked to have a filter specific to `Search Forms`, so this will be OBE in the next commit